lib/exports/colors-new/__snapshots__/colors.less.test.ts.snap

```diff
@@ -0,0 +1,752 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`colors > should output all the css generated by the new colors/theming generation mixins 1`] = `
+"body {
```

Comment:
In `body`, we have the baseline state alias (error, success, etc) and component-specific theme variables defined.